index.html

```diff
@@ -60,10 +60,10 @@ <h2>The 140 Hour Movie Watching Challenge</h2>
             <!-- Featured Post -->
             <article class="post featured">
                 <header class="major">
-                    <h2><a href="#">Performance Art on a Minscule Scale</a></h2>
```

Comment:
meniskley

Reply:
u r a jerk josiahroberts

Reply:
whatever - I already fixed it - *_IN ENGLISH_*